lib/test/index.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8">
-  <title>Ramda Tests</title>
-  <link rel="stylesheet" href="mocha.css" />
-</head>
-<body>
-  <h1>Tests for ramda</h1>
-  <div id="mocha"></div>
-  <script src="lib/test/expect.js"></script>
-  <script src="mocha.js"></script>
-  <script src="assert-shim.js"></script>
-  <!--[if lt IE 9]>
-    <script src="../lib/sauce/ie8-shim.js"></script>
-  <![endif]-->
-  <script>mocha.setup('bdd')</script>
-  <script src="bundle.js"></script>
-  <script src="../sauce/mocharun.js"></script>
-</body>
-</html>
```

Comment:
The unused files in __lib/test__ threw me off as I tried to make sense of all the various test scripts. I'd prefer to have just one way to run browser tests: `npm run testem`. In my case this runs the test suite in Chrome, Firefox, Opera, and Safari in less than a minute. It's neat!